### PR TITLE
Redesign the compass application.

### LINKED ISSUE #1: Indicate user action during initial compass calibration
Upon first use of the compass application e.g. `Calibrated: 0`, the compass is not animated due to not yet being calibrated, but there is no indication to the user whether or not any input is required.

It would be helpful to indicate some notification, such as "Please wait while the compass is calibrated". Otherwise, as a user, I either assume that I am doing something wrong or that the application is broken.

### PR DESCRIPTION
**Pull request by MagneFire**
This PR changes the look and feel of the compass application. This should make it look more in line with the AsteroidOS design scheme.

| Old | New (round) | New (rectangular) |
| ----|----|----|
| nar-old | nar | bel |

Additionally the `StatusPage` is used to prompt for a calibration instead of label:
nar-mar
See also the relevant PR: https://github.com/AsteroidOS/qml-asteroid/pull/20/files

This should also fix https://github.com/AsteroidOS/asteroid-compass/issues/1.

### CODE REVIEW

File: src/main.qml [hunk 1]
```diff
@@ -37,22 +40,86 @@ Application {
...
+            rotation: -app.rotation
+            Repeater {
+                id: outerRing
+                anchors.fill: parent
+                model: 8
+                Label {
+                    property var angle: (index / outerRing.count) * 2 * Math.PI
+                    rotation: app.rotation
+                    color: index == 0 ? "#c2620c" : "white"
+                    text: index == 0 ? "N" :
```
Comment (moWerk) [inline src/main.qml]: This could be an array of `[N, NE, E ...` where the index iterates through said array?

File: src/main.qml [hunk 2]
```diff
@@ -37,22 +40,86 @@ Application {
...
+                        index == 1 ? "NE" :
+                        index == 2 ? "E" :
+                        index == 3 ? "SE" :
+                        index == 4 ? "S" :
+                        index == 5 ? "SW" :
+                        index == 6 ? "W" :
+                        "NW"
+                    anchors.centerIn: parent
```
Comment (moWerk) [inline src/main.qml]: Marginal, but i can better read:
anchors {
    centerIn: parent
    verticalCenterOffset: -Math.cos(angle) * Dims.l(40)
    horizontalCenterOffset: Math.sin(angle) * Dims.l(40)
}

**Review by moWerk — CHANGES_REQUESTED** [review]
Love your compass design!

**Comment by MagneFire** [comment]
Thanks for the feedback, this reduces the code size even more :smile: 
(I was already proud to reduce it from whatever this garbage is https://github.com/MagneFire/asteroid-compass/commit/5dcbe4cf77daabff5ea34f99846fd8ac728c27f9 :sweat_smile: )

**Review by moWerk — APPROVED** [review]

**Comment by FlorentRevest** [comment]
Love this, thanks!

**Pull request merged**